apps/data-types.md

```diff
@@ -4,111 +4,145 @@ key: apps-data-types
 title: Data types
 ---
 
-In the following the JSON supported data types are listed.
+# address
 
-# Number
+This object is used for the attributes of shippingAddress and billingAddress.
```

Comment:
missing attributes in address:
- company
- salutation
- streetDetails
- state
- title
- vadId